### DISCUSSION TITLE
Request: ページツリー（Page Tree）からユーザー項目を削除できるようになりますか？

### DISCUSSION THREAD

**Discussion by KanoeGitHub** [growilabs/growi#7730] [Ideas / Requests] [answered]
## Informations
ページツリーからユーザー項目を削除する方法が見当たりません。  
これはGrowiの開発ポリシーによるものでしょうか？
Userページは特定のグループのみ、またはプロジェクトように運用する場合には重要性が高くない場合があると思います。
さらに、ページツリーは現在アルファベット順に項目が並ぶため、プロジェクトごとのトピックが並んでいる中に、突然Userページが出現することにもなります。
これを表示させないようにする設定か、もしくはUser項目を削除できるような機能が欲しいと考えています。

**Comment by orlea** [comment]
現状ユーザーをGROWIから削除してもユーザーページは残るように見受けられます(私の方で試した限りはそうでした)。

https://github.com/weseek/growi/discussions/6467 このRequestのように削除できるようになるか、ユーザー削除と同期してページも削除されるか、`/user/` が無効になるような設定が実装されると嬉しいです。

**Comment by yuki-takei** [comment]
現在実装中です。

https://redmine.weseek.co.jp/issues/105325

**Comment by jam411** [comment]
- [v6.2.0](https://github.com/weseek/growi/tree/v6.2.0) でユーザー新規登録時に同じ username のユーザーホームページがある場合はそれを一度削除して作り変えるように仕様変更されました。
また、ユーザー削除時にユーザーホームページも一緒に削除する設定ができるようになりました。
詳しくはアップグレードガイドをご覧ください。
https://docs.growi.org/ja/admin-guide/upgrading/62x.html#growi-v6-2-x-%E3%81%B8%E3%81%AE%E3%82%A2%E3%83%83%E3%83%95%E3%82%9A%E3%82%AF%E3%82%99%E3%83%AC%E3%83%BC%E3%83%88%E3%82%99

**Comment by yuki-takei** [comment, marked as answer]
[v6.2.4](https://github.com/weseek/growi/tree/v6.2.4) で、設定により削除済みユーザーのユーザーホームページを通常ページと同じように削除できるようになりました。
